table/block_based_table_builder.cc

```diff
@@ -374,6 +399,8 @@ void BlockBasedTableBuilder::Add(const Slice& key, const Slice& value) {
       }
     }
 
+    // Note: PartitionedFilterBlockBuilder requires adding to filter_block to be
```

Comment:
... requires key being added to filter builder after being added to index builder.